src/components/ResultDisplay.module.css

```diff
@@ -28,3 +30,20 @@
 	padding: 1rem;
 	overflow: auto;
 }
+
+.sortWidget {
```

Comment:
Request: `cursor: pointer`? Buttons don't have it by default.

Reply:
✅done!